### ISSUE TITLE
VMware.vSphere.SsoAdmin Group membership

### ISSUE THREAD

**Issue by rschitz** [vmware/PowerCLI-Example-Scripts#440]
Hi, thanks for this great module. Is it possible to get group membership to check if a specific user is member of a specific group (i.e. list the Administrators group members)

**Comment by dmilov** [comment]
Hello, no at the moment. I can extend the `Get-PersonUser` cmdlet with `Group` parameter to return the members of the group.

**Comment by dmilov** [comment]
Implemented this one. I'll implement two more requests before publishing the module.

**Comment by dmilov** [comment]
Added `-Group`  parameter to `Get-SsoPersonUser`. It allows getting members of a group the following way:
```powershell
Get-SsoGroup -Name 'Administrators' -Domain 'vsphere.local' | Get-SsoPersonUser
```

Fixed in version 1.2.3 available on GitHub and PSGallery

**Comment by rschitz** [comment]
Thanks a lot @dmilov !
